src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadata.java

```diff
@@ -188,11 +192,15 @@ public void onFailure(Throwable t) {
     );
   }
 
+  @Override
   public void setAnnotationsVisible(boolean visible) {
```

Comment:
Any change to annotation visibility should go through presenter, since it needs to store current visibility in a local instance variable